schemas/telemetry/new-profile/new-profile.4.schema.json

```diff
@@ -477,6 +486,12 @@
         },
         "system": {
           "properties": {
+            "appleModelId": {
```

Comment:
This field says it can be null in [the docs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html)